efficientdet/efficientdet_arch.py

```diff
@@ -282,6 +282,108 @@ def class_net(images,
   return classes
 
 
+class ClassNet(tf.keras.layers.Layer):
+    def __init__(self,
+                 config,
+                 name='class_net'):
+
+        super(ClassNet, self).__init__(name=name)
```

Comment:
`tf.keras.layers.Layer` can take multiple `**kwargs`, such as `trainable` and `dtype`, including some optional ones for backwards compatibility. I recommend passing them to the parent-class constructor like this:

```
    def __init__(self, config, name='class_net', **kwargs):
        super(ClassNet, self).__init__(name=name, **kwargs)
```

https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer.